docs/Using Fleet/Vulnerability-Processing.md

```diff
@@ -19,91 +13,21 @@ Vulnerable software can have one or more vulnerabilities (CVEs).
-
-### Linux hosts
-
-First, we determine what Linux distributions are part of your fleet (keep in mind that there will
-be a small delay between the time a new Linux hosts is added and the time the host is 'detected'). We then
-use
-that information to determine what OVAL definitions need to be downloaded and parsed - you can find
-a list of all the OVAL definitions we use
-[here](https://github.com/fleetdm/nvd/blob/master/oval_sources.json). OVAL definitions will be
-refreshed on a daily basis.
-
-Finally, we look at the software inventory of each host and execute the assertions contained in the
-corresponding OVAL file - any match is reported using the same channels as with Windows/Mac OS vulnerabilities
+[Learn more](https://github.com/fleetdm/fleet/blob/main/docs/Contributing/Vulnerability-processing.md) about performance and how it works for different platforms.
 
 ## Coverage
 
-For Windows/Mac OS Fleet attempts to detect vulnerabilities for installed software that falls into the following categories (types):
-
-### Apps
-
-- macOS
-- Windows
-
-### Browser plugins
-
-- macOS, Windows
-  - Chrome extensions
-  - Firefox extensions
+Fleet attempts to detect vulnerabilities for installed software that falls into the following categories (types):
 
-### Packages
-
-- macOS
-  - Python
-  - Packages installed using Homebrew
-
-- Windows
-  - Python
-  - Atom
-  - Packages installed using Chocolatey
-
-For Linux, we adhere to whatever is defined in the OVAL definitions, except for:
-
-- Kernel vulnerabilities.
-- Vulnerabilities involving configuration files.
-
-As of right now, the following distributions are supported:
-
-- Ubuntu
-- RHEL based distros (Red Hat, CentOS, Fedora, and Amazon Linux)
-
-As of right now, only app names with all ASCII characters are supported. Apps with names featuring
-non-ASCII characters, such as Cyrillic, will not generate matches.
-
-The ingestion of software varies per platform. For each platform, we run an [osquery query](#ingesting-software-lists-from-hosts) to ingest software.
+| Type                | macOS                                      | Windows                                          | Linux            |
+| ------------------- | ------------------------------------------ | ------------------------------------------------ | ---------------- |
+| Apps                | Yes                                        | Yes                                              | --               |
+| Browser plugins     | Chrome extensions, Firefox extensions      | Chrome extensions, Firefox extensions            | --               |
+| Packages            | Python, Homebrew  | Python, Atom, Chocolatey | Adhere to whatever is defined in the OVAL definitions, except for kernel vulnerabilities and vulnerabilities involving configuration files. Supported distributions: <ul><li>Ubuntu</li><li>RHEL based distros (Red Hat, CentOS, Fedora, and Amazon Linux)</li></ul> |
```

Comment:
What is defined in the OVAL definitions? I think we should spell this out for the user if we can.

Reply:
@noahtalerman I added a link to [OVAL sources](https://github.com/fleetdm/nvd/blob/master/oval_sources.json).